lib/id3c/cli/command/etl/redcap_det.py

```diff
@@ -229,9 +229,9 @@ def insert_fhir_bundle(db: DatabaseSession, bundle: dict) -> None:
     LOG.info(f"Inserted FHIR document {fhir.id} «{bundle['id']}»")
 
 
-def mark_loaded(db: DatabaseSession, det_id: int, etl_id: dict) -> None:
+def mark_loaded(db: DatabaseSession, det_id: int, etl_id: dict, bundle_uuid: str) -> None:
     LOG.debug(f"Marking REDCap DET record {det_id} as loaded")
-    mark_processed(db, det_id, {**etl_id, "status": "loaded"})
+    mark_processed(db, det_id, {**etl_id, "status": "loaded", "fhir_bundle_uuid": bundle_uuid})
```

Comment:
Minor thing, but I'd name the field `fhir_bundle_id` since `id` is the name of the field in the Bundle itself. That the value is a UUID is an implementation detail for us; the Bundle id is allowed to be other values by the spec.